http/adapter.go

```diff
@@ -43,6 +47,9 @@ func (as *adapterService) Publish(ctx context.Context, token string, msg *messag
 		return err
 	}
 	msg.Publisher = thid.GetValue()
+	fmt.Println()
+	fmt.Println("Called as.pubslisher.publish")
+	fmt.Println()
```

Comment:
Do we need these print line?

Reply:
This must be while merging, I'll remove them asap!